### PR TITLE
add dForce USDx and LENDF.ME

### PR DESCRIPTION
**Pull request by daboooooo**

### CODE REVIEW

**Comment by hboon** [comment]
@daboooooo thank you for your pull request. We are currently having problems with getting the index of Dapps on the iOS app store, and have removed it completely. So while we might merge your PR, you will not see it in the app anytime soon. This isn't an issue on Android.

**Comment by hboon** [comment]
@daboooooo https://lendf.me isn't responding

**Comment by daboooooo** [comment]
@hboon The url of lendf.me has been updated. Thanks for your mention.

File: AlphaWallet/Browser/ViewModel/Dapps.swift [hunk 1]
```diff
@@ -56,7 +56,11 @@ enum Dapps {
         Dapp(name: "Humanity", description: "Human Identity on Ethereum", url: "https://humanitydao.org", cat: "Social Media"),
         Dapp(name: "DEX.AG", description: "Trade cryptoassets at the best price", url: "https://dex.ag", cat: "Exchange"),
         Dapp(name: "Totle Swap", description: "Totle automatically finds and acquires the best price across decentralized exchanges for ERC-20 swaps", url: "https://swap.totle.com", cat: "Exchange"),
-        Dapp(name: "ATS Bridge", description: "ATS/ATS20 bridge for self transfers of ATS to ATS20", url: "https://bridge.artis.network/", cat: "Tool")
+        Dapp(name: "ATS Bridge", description: "ATS/ATS20 bridge for self transfers of ATS to ATS20", url: "https://bridge.artis.network/", cat: "Tool"),
+        Dapp(name: "dForce USDx", description: "A decentralized and synthetic indexed stablecoin with interest bearing capability", url: "https://usdx.dforce.network/", cat: "Finance"),
+        Dapp(name: "LENDF.ME", description: "Algorithmic money markets for dForce USDx", url: "https://www.lendf.me/", cat: "Finance")
+]
```
Comment (James-Sangalli) [inline AlphaWallet/Browser/ViewModel/Dapps.swift]: This is breaking the build, please remove it and rebase your commits into one. Cheers

**Pull request closed without merging**